### PR TITLE
Week4 : add Map API

### PR DESCRIPTION
**Pull request by raniarusly**
Screenshot 2020-07-28 at 2 59 54 PM

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -155,3 +155,17 @@ async function checkLogin(){
     document.getElementById("login-link").href = details.loginUrl;
   }
 }
+
+/** Creates a map and adds it to the page. */
+function initMap() {
+  const nightMap = new google.maps.StyledMapType(nightVersion, {name: 'Night'});
```
Comment (MostafaGabriel) [inline portfolio/src/main/webapp/script.js:161]: Can you please add a screenshot to the night version of the map? :) I'm excited to see what does it look like :D

File: portfolio/src/main/webapp/nightVersion.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  {
+    "featureType": "water",
+    "elementType": "labels.text.stroke",
+    "stylers": [
+      {
+        "color": "#17263c"
+      }
+    ]
+  }
+]
```
Comment (KaziZubaer) [inline portfolio/src/main/webapp/nightVersion.js:162]: Nit: Newline :)

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -155,3 +155,17 @@ async function checkLogin(){
...
+  const map = new google.maps.Map( document.getElementById("map"), {
```
Comment (KaziZubaer) [inline portfolio/src/main/webapp/script.js:162]: Nit: Both of these variables have suffix `map` that kinda indicates that they represent the same type of object - which is not true I guess. Could you please change the name of the first variable a bit? I guess it represents some _type_ or _style_ and hence you can probably rename it to `nightMapType` or `nightMapStyle`? 

This is a rule of thumb that the last part of a variable name should say what the variable is basically.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (raniarusly) [reply portfolio/src/main/webapp/script.js:161]: Hi this is what the map looks like! and yes I make it to be centered in Jakarta, Indonesia :D
standard (day) map
image


night map
Screenshot 2020-07-28 at 6 54 35 PM

Comment (MostafaGabriel) [reply portfolio/src/main/webapp/script.js]: Looks great! :D Thanks Rania :)

**Pull request merged**